apps/vault/test/vault_shared.js

```diff
@@ -295,9 +295,9 @@ module.exports = (vaultName, { accounts, artifacts, web3 }) => {
 
             const vaultBalance = await token.balanceOf(vault.address)
             assert.equal(vaultBalance, 5, 'token accounting should be correct')
-            assert.deepEqual(
-              await vault.balance(token.address),
-              vaultBalance,
+            assert.equal(
```

Comment:
Ah, it's better to use the string comparison here—that way, the test failure will show both the expected and actual values in the output.

Reply:
Oh nice, I didn't realize that. Thanks